src/librustc_error_codes/error_codes/E0703.md

```diff
@@ -0,0 +1,18 @@
+Invalid ABI(Application Binary Interface) used in the code.
+
+Erroneous code example:
+
+```compile_fail,E0703
+extern "invalid" fn foo() {} //~ ERROR
+
+fn main() { }
```

Comment:
Btw, no need to display it in the docs, you can hide the line by starting it with "# ". So please rewrite the code example like this:

```
extern "invalid" fn foo() {} // error!
# fn main() {}
```